style.scss

```diff
@@ -0,0 +1,230 @@
+@import "node_modules/codemirror/lib/codemirror.css";
+@import "node_modules/codemirror/addon/hint/show-hint.css";
+@import "node_modules/codemirror/addon/lint/lint.css";
+@import "node_modules/codemirror/addon/scroll/simplescrollbars.css";
+
+* {
+  box-sizing: border-box;
+}
+
+.editor {
+  float: left;
+  width: 50%;
+  margin: 0;
+  padding: 2%;
+  height: 100%;
+  background-color: gainsboro;
+  border: gainsboro;
+  border-radius: 10px;
+  .shortcut{
+    color: gray ;
+    font-size: 14px ;
+    display: inline;
+    margin: 0 5px;
+  }
+  .line-error {
+    background: #fbc2c4 !important;
+    color: #8a1f11 !important;
+  }
+  .error-marker {
+    color: black;
+    width: 10px !important;
+    background-color: #ff0000;
+  }
+
+  .error-marker {
+    .error-message {
+      display: none;
+      position: absolute;
+      background-color: #ddd;
+      border: 1px solid #999;
+      padding: 6px;
+      width: 140px;
+      left: 15px;
+      top: -1em;
+    }
+  }
+
+  .error-marker:hover {
+    .error-message {
+      display: block;
+    }
+  }
+}
+.save_load {
+  width: 100%;
+  height: 100%;
+  float: left;
+}
+// #ls {
```

Comment:
if you don't need the stray code please make sure to clean up

Reply:
Got it! thanks